### ISSUE TITLE
In lines per author graphs, sorting of authors is not correct

### ISSUE THREAD

**Issue by pulkomandy** [vifactor/repostat#127]
Authors are sorted by commit count instead of line count in this graph. As a result, the legend is in the same order as the graph below, and in the popup with numbers you can see that the items are not sorted by number of lines as would be expected

**Comment by vifactor** [comment]
I would think of replacing this graph with something smarter as contribution is not only how many lines added, but also how many lines removed. Perhaps, it is worth to remove this plot and plot streamgraph using not number of commits, but `modifications_count = lines_added + lines_removed`.

Sorting there I would say is less important.

**Comment by pulkomandy** [comment]
Yes, I find this graph not so useful in the current form. Maybe number of "owned" line (as in git blame) would be more interesting, but not as easy to collect.

I'm trying to think of something nice to show added vs removed lines. Maybe a barchart or stacked graph with bars above/below 0. I may experiment with that.

**Comment by vifactor** [comment]
> Maybe number of "owned" line (as in git blame) would be more interesting, but not as easy to collect.

this is technically easy to collect, but extremelly slow. for big repos with thousands of files, it will simply be impossible to wait that long (even if enable multiprocessing CPU cores).

> I'm trying to think of something nice to show added vs removed lines

You may try to implement same graphs as github does when you go to "contributors" stats, I do not now whether it will look good for even 5-7 authors on one plot
